src/libs/ReportUtils.ts

```diff
@@ -623,6 +635,13 @@ function getReport(reportID: string): OnyxEntry<Report> {
     return ReportConnection.getAllReports()?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
 }
 
+/**
+ * Returns the report
+ */
+function getReportNameValuePair(reportID?: string): OnyxEntry<ReportNameValuePairs> {
```

Comment:
Is there a reason you called this `getReportNameValuePair` instead of `getReportNameValuePairs`? (With the s) Otherwise, I think it's better to call this `getReportNameValuePairs`